controllers/dbconfig.php

```diff
@@ -1,13 +1,14 @@
 <?php
     class DbconfigController extends ControllerBase {
         protected $method = 'create';
-        protected function dbInit() {}
+        // protected function dbInit() {} // deprecated dunno why this was here. 

```

Comment:
This was here because the `dbInit()` function is called by all controllers through ControllerBase, so it must be substituted for a dummy one if the database isn't set up. @bugos can tell you more.